### PR TITLE
OCM-7141 | ci: Automate 72508/72521

### PR DESCRIPTION
**Pull request by radtriste**
depends on https://github.com/terraform-redhat/terraform-provider-rhcs/pull/610

**What this PR does / why we need it**:

Which issue(s) this PR fixes
https://issues.redhat.com/browse/OCM-7141

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

**Comment by radtriste** [comment]
<details>
<summary>72508</summary>
Will run 1 of 71 specs
time="2024-04-09T15:22:51+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-09T15:22:51+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-09T15:22:54+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
Verify cluster can not apply changes to security group - [id:69145] [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:243
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSStime="2024-04-09T15:22:56+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
time="2024-04-09T15:22:56+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
time="2024-04-09T15:22:56+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-09T15:23:00+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/aws/vpc"
time="2024-04-09T15:23:01+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config with args [-var spec_vm_dirty_ratios=[65,65,65] -var spec_priorities=[10,10,10] -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=3]"
time="2024-04-09T15:23:05+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
time="2024-04-09T15:23:05+02:00" level=info msg="Retrieved tuning configs: [tc-0 tc-1 tc-2]"
time="2024-04-09T15:23:05+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var machine_type=m5.2xlarge -var replicas=3 -var autoscaling_enabled=false -var auto_repair=true -var tags={\"tag1\":\"test_tag1\",\"tag2\":\"test_tag2\"} -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name=np-72504-dr -var url=https://api.stage.openshift.com -var subnet_id=subnet-0213b895dd34ab887 -var tuning_configs=[\"tc-0\",\"tc-1\",\"tc-2\"]]"
time="2024-04-09T15:23:10+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp with args [-var url=https://api.stage.openshift.com -var subnet_id=subnet-0213b895dd34ab887 -var tuning_configs=[\"tc-0\"] -var replicas=3 -var autoscaling_enabled=false -var auto_repair=true -var tags={\"tag1\":\"test_tag1\",\"tag2\":\"test_tag2\"} -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name=np-72504-dr -var machine_type=m5.2xlarge]"
time="2024-04-09T15:23:14+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/machine-pools/hcp"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var replicas=3 -var tags={"tag1":"test_tag1","tag2":"test_tag2"} -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var url=https://api.stage.openshift.com -var autoscaling_enabled=false -var subnet_id=subnet-0213b895dd34ab887 -var tuning_configs=["tc-0"] -var auto_repair=true -var name=np-72504-dr -var machine_type=m5.2xlarge
time="2024-04-09T15:23:17+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=3 -var spec_vm_dirty_ratios=[65,65,65] -var spec_priorities=[10,10,10]
•SSSSSS

Ran 1 of 71 Specs in 30.146 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 69 Skipped
PASS

</details>

<details>
<summary>72521</summary>
Will run 1 of 71 specs
time="2024-04-09T14:32:10+02:00" level=info msg="Loaded cluster profile configuration from profile rosa-hcp-ad : map[admin_enabled:false autoscale:true byok:true byovpc:true channel_group:candidate cloud_provider:aws cluster_type:rosa-hcp compute_machine_type:m5.2xlarge etcd_encryption:true fips:false kms_key_arn:true labeling:false major_version:4.15 multi_az:true oidc_config:managed private:false product_id:rosa proxy:true region:us-west-2 sts:true tagging:true unified_acc_role_path: version: version_pattern:latest zones:a,b,c]"
time="2024-04-09T14:32:10+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
time="2024-04-09T14:32:13+02:00" level=info msg="Running terraform output against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters/rosa-hcp"
SSSSSSSSSSSSSSSS
------------------------------
P [PENDING]
Verify cluster can not apply changes to security group - [id:69145] [NonHCPCluster]
/home/tradisso/projects/openshift/terraform-provider-rhcs/tests/e2e/verfication_post_day1_test.go:243
------------------------------
SSSSSSSSSSSSSSSStime="2024-04-09T14:32:15+02:00" level=info msg="Running terraform init against the dir /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
time="2024-04-09T14:32:15+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config with args [-var spec_vm_dirty_ratios=[65] -var spec_priorities=[20] -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=1]"
time="2024-04-09T14:32:18+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config with args [-var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=2 -var spec_vm_dirty_ratios=[65,45] -var spec_priorities=[20,10] -var cluster=2agke2gr307vepjh32rp38rd6997l3ed]"
time="2024-04-09T14:32:21+02:00" level=info msg="Running terraform apply against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config with args [-var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=2 -var spec_vm_dirty_ratios=[55,45] -var spec_priorities=[1,10] -var cluster=2agke2gr307vepjh32rp38rd6997l3ed]"
time="2024-04-09T14:32:25+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=2 -var spec_vm_dirty_ratios=[55,45] -var spec_priorities=[1,10]
time="2024-04-09T14:32:28+02:00" level=info msg="Running terraform destroy against the dir: /home/tradisso/projects/openshift/terraform-provider-rhcs/tests/tf-manifests/rhcs/tuning-config"
args:  /usr/bin/terraform destroy -auto-approve -no-color -var cluster=2agke2gr307vepjh32rp38rd6997l3ed -var name_prefix=tc -var url=https://api.stage.openshift.com -var tc_count=2 -var spec_vm_dirty_ratios=[55,45] -var spec_priorities=[1,10]
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 71 Specs in 18.540 seconds
SUCCESS! -- 1 Passed | 0 Failed | 1 Pending | 69 Skipped
PASS

</details>

File: tests/e2e/tuning_config.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	It("can create/edit/delete - [id:72521]",
+		ci.Day2, ci.High, ci.FeatureTuningConfig, func() {
+
+			tcCount := 1
+			specVMDirtyRatios := []int{65}
+			specPriorities := []int{20}
+			By("Create one tuning config")
+			tcArgs = &exec.TuningConfigArgs{
+				Cluster:           clusterID,
+				NamePrefix:        "tc",
```
Comment (xueli181114) [inline tests/e2e/tuning_config.go:45]: How about has case ID in the Name prefix? So if any resource leak we can know which case caused that.

File: tests/e2e/tuning_config.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+			}
+			_, err := tcService.Apply(tcArgs, false)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Verify tuning config")
+			tcsResp, err := cms.ListTuningConfigs(ci.RHCSConnection, clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(tcsResp.Size()).To(Equal(tcCount))
+			tc := tcsResp.Items().Get(0)
+			Expect(tc.Name()).To(Equal("tc-0"))
```
Comment (xueli181114) [inline tests/e2e/tuning_config.go:58]: The name here is not updated

Comment (radtriste) [reply tests/e2e/tuning_config.go]: updated

**Pull request merged**